packages/nextjs/components/scaffold-stark/Address.tsx

```diff
@@ -37,6 +36,12 @@ const blockieSizeMap = {
   "3xl": 15,
 };
 
+const TypedCopyToClipboard = CopyToClipboard as unknown as React.FC<{
```

Comment:
Here I added the TypedCopyClipboard again, remove it, and check that the rest is correct in the file

https://github.com/Scaffold-Stark/scaffold-stark-2/blob/main/packages/nextjs/components/scaffold-stark/Address.tsx